components/speedreader/rust/lib/src/readability/src/extractor.rs

```diff
@@ -1,133 +1,99 @@
 use crate::dom;
+use crate::scorer;
+use crate::util;
 use html5ever::tendril::StrTendril;
-use html5ever::tendril::TendrilSink;
 use html5ever::tree_builder::TreeSink;
 use html5ever::tree_builder::{ElementFlags, NodeOrText};
-use html5ever::{parse_document, serialize};
 use html5ever::{LocalName, QualName};
-use markup5ever_rcdom::RcDom;
-use markup5ever_rcdom::SerializableHandle;
-use scorer;
-use scorer::{Candidate, Title};
-use std::cell::Cell;
-use std::collections::BTreeMap;
+use kuchiki::NodeRef as Handle;
+use kuchiki::Sink;
+use regex::Regex;
+use scorer::{Title, TopCandidate};
 use std::collections::HashMap;
 use std::default::Default;
-use std::io::Read;
-use std::path::Path;
-use std::rc::Rc;
 use std::str::FromStr;
 use url::Url;
+use util::StringUtils;
+
+const NUM_TOP_CANDIDATES: usize = 5;
```

Comment:
Why is `NUM_TOP_CANDIDATES ` == 5? An inline comment explaining the reason would be helpfult.